dsc/quickstarts/gpo-quickstart.md

```diff
@@ -0,0 +1,66 @@
+---
+ms.date:  07/09/2019
+keywords:  dsc,gpo,powershell,configuration,setup
+title:  Quickstart - Convert Group Policy into DSC
+---
+
+> Applies To: Windows PowerShell 4.0, Windows PowerShell 5.0
+
+# Quickstart: Convert Group Policy into DSC
+
+You can generate a DSC configuration from a Group Policy or Azure Security Center baseline. The [BaselineManagement](https://www.powershellgallery.com/packages/BaselineManagement/2.8.8000)
```

Comment:
Is there a reason to link to that specific version? If you leave the version off of the URL it will default to latest. The latest version is 2.9.0.